### PR TITLE
UI improvement

### PR DESCRIPTION
**Pull request by zekth**
fixing #575

capture d ecran 2018-10-01 a 22 04 37

Summary has been move to a component in Configuration / About. Design of it is really RAW at the moment but it could host more infos i think.

### CODE REVIEW

File: jsx/configuration/main.jsx [hunk 1]
```diff
@@ -12,7 +13,8 @@ const components = {
   telemetry: <Telemetry />,
   authentication: <Auth />,
   admin: <Admin />,
-  errors: <Errors />
+  errors: <Errors />,
+  about: <About />
```
[the remark this answers is not in the document]
Comment (zekth) [reply jsx/configuration/main.jsx]: Starting to make a difference between user agents and not using responsive will be a mess in my opinion. i'd rather hide the component on mobile device.

**Comment by zekth** [comment]
Conditionnal component rendering based on user agent is not really a solution, i'd prefer to hide them with responsive feature.
I'll put back the Summary in footer but do i keep the About section? Which is kind of standard for any application.

**Comment by zekth** [comment]
Added better UI for about section:
2018-10-02 10_40_35-reef-pi

**Comment by ranjib** [comment]
@Ranthalion let me know where you stand on this. I am ok with merging this

**Pull request merged**